packages/babel-plugin-minify-simplify/src/index.js

```diff
@@ -333,7 +333,12 @@ module.exports = ({ types: t }) => {
       Function: {
         enter: earlyReturnTransform,
 
-        exit(path) {
+        exit(path, state) {
+          const multiPass = state.opts.multiPass === false ? false : true;
```

Comment:
For this (putting everything in one line and just one 2-level-deep option) it looks cryptic, actually using this is much clearer sometimes, We could use default destructuring assignment when we have more options.